src/app/components/ReadTime/index.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReadTime from "."; 
+import readme from './README.md';
+
+
+
+const Component = () => {  return (
+    <ReadTime readTimeValue={1} />
+  );
+}  
+
+export default {
+  title: 'Components/ReadTime',
+  Component,
+  parameters: {
+    docs: { readme },
+  },
+};
+
+export const Example = () => <Component />;
```

Comment:
I wonder if we should add another example for more than 1 minutes - which will be helpful once we've added translations